src/raftstore/store/fsm/peer.rs

```diff
@@ -2084,6 +2215,8 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
         if !self.fsm.peer.is_leader() {
             self.ctx.raft_metrics.invalid_proposal.not_leader += 1;
             let leader = self.fsm.peer.get_peer_from_cache(leader_id);
+            self.fsm.group_state = GroupState::Chaos;
```

Comment:
Is it really necessary?

Reply:
Yes. It usually means leader is not responding to client's requests, then a new leader needs to stand up.

Reply:
Go it. It's the new machanism to avoid waiting 5 minutes to start election. Thanks!